docker/nginx/conf.d/include/ratelimited

```diff
@@ -0,0 +1,6 @@
+# Add a list of IPs here that should be severely rate limited on upload.
+# Note that it is possible to add IP ranges as well as the full IP address.
+#
+# Examples:
+# 192.168.0.0/24 1;
+# 79.85.222.247 1;
```

Comment:
This looks fine, I am just not super happy that we will be changing a committed file on the servers. I usually want to keep the repo file untouched on the servers if not necessary - ex. using `docker-compose.override.yml` instead of directly modifying `docker-compose.yml`.

Reply:
Hm I didn't think of this. If you include a non existing file in nginx, would it complain you think? Because if not we can just remove this file. I only added it to have an example of the ip ranges, but we can move these comments to where we include the file.

Reply:
We might be able to do something like this:
```
geo $limit {
	default 0;
	include /etc/nginx/conf.d/include/blocklist/*.ips;
}
```
then rename the above file (and move it) to `/etc/nginx/conf.d/include/blocklist/example.ips;`
and then define `/etc/nginx/conf.d/include/blocklist/skynet.ips;` on the server

Reply:
I like it but I would actually drop the `.ips` extension, it doesn't tell much but I would definitely move that file you had with readme and add info "create a file and include ips delimitered by new lines like this [example]" and we should be fine.